AudioConverter/converter.py

```diff
@@ -165,11 +165,15 @@ def get_audio_files(input_path: pathlib.Path) -> Sequence[pathlib.Path]:
     Recursively get audio files within the input_path.
     """
     audio_files = []
-    for input_file in input_path.iterdir():
-        if input_file.is_file() and input_file.suffix.lower() in AUDIO_EXTENSIONS_SET:
-            audio_files.append(input_file)
-        elif input_file.is_dir() and not input_file.is_symlink():
-            audio_files.extend(get_audio_files(input_file))
+    if input_path.is_file():
+        if input_path.suffix.lower() in AUDIO_EXTENSIONS_SET:
+            audio_files.append(input_path)
+    else:
+        for input_file in input_path.iterdir():
+            if input_file.is_file() and input_file.suffix.lower() in AUDIO_EXTENSIONS_SET:
+                audio_files.append(input_file)
+            elif input_file.is_dir() and not input_file.is_symlink():
+                audio_files.extend(get_audio_files(input_file))
```

Comment:
I think the recursive method can just handle the base case of file  simplify some of the logic. Something like,
```
def get_audio_files(input_path: pathlib.Path) -> Sequence[pathlib.Path]:
  if input_path.is_file() and input_path.suffice.lower() in AUDI_EXTENSIONS_SET:
    return [input_path]
  elif not input_path.is_dir():
    return []
  audio_files = []
  for input_file in input_path.iterdir():
    audio_files.extend(get_audio_files(input_file))
  return audio_files
```